### DISCUSSION TITLE
How can I change my password?

### DISCUSSION THREAD

**Discussion by teon** [buttondown/discussions#39] [Q&A] [answered]
I have logged in by google - but the email attached to the account is no longer managed by google. So I can't login by google.
When I enter password reset and the email - I'm not getting the password reset link.

So how can I reset my password?

**Comment by jmduke** [comment, marked as answer]
Great question with an annoying answer. (We'll add this to the docs.) Reach out to Support (support@buttondown.email): they'll manually change your account to switch it over from Google-based to UN/PW-based.

**Comment by teon** [comment]
@jmduke support isn’t much of a help.

I think i will drop this service.
